lib/dal/src/workspace_snapshot/graph/v4/component.rs

```diff
@@ -102,3 +107,20 @@ impl WorkspaceSnapshotGraphV4 {
         Ok(results)
     }
 }
+
+impl ComponentExt for WorkspaceSnapshotGraphV4 {
+    fn root_attribute_value(
+        &self,
+        component_id: ComponentId,
+    ) -> WorkspaceSnapshotGraphResult<AttributeValueId> {
+        let component_index = self.get_node_index_by_id(component_id)?;
```

Comment:
doesn't this functionality already exist? just wondering why an extension trait. My question here is because this means these will have to be implemented twice now (one for this, one for the split graph)

Reply:
Oh, we can remove this one I think--we stopped using it when we moved the edge to point to AttributeValue instead of Component.

Reply:
Removed.